src/connectors/bandcamp.js

```diff
@@ -139,15 +139,7 @@ function initPropertiesForCollectionsPlayer() {
 
 	Connector.trackArtSelector = '.now-playing img';
 
-	Connector.getOriginUrl = () => {
-		const buyNowAnchor = document.querySelector('.buy-now a');
-		if (buyNowAnchor === null) {
-			Util.debugLog('Failed to resolve originUrl');
-			return document.location.href;
-		}
-
-		return buyNowAnchor.href.split('?')[0];
-	};
+	Connector.getOriginUrl = () => getOriginUrl('.buy-now a');
```

Comment:
This should cover the 'collections' tab as well as the 'wishlist' tab (which are on the same page)

```suggestion
	Connector.getOriginUrl = () => getOriginUrl('.playing .collection-title-details a, .playing .buy-now a');
```

Reply:
Looks like we can even simplify it

Reply:
Actually it didn't work and applied your full fix - thanks :pray: